### ISSUE TITLE
Setting up shipping for member swag

### ISSUE THREAD

**Issue by carogea** [wherebyus/general-tasks#1107]
Waiting on a credit card to pay for grant expenses; need to make a decision about Drop Shipping

DOD: shipping is set up, decision is made about how to move forward

**Comment by michaelschofield** [comment]
I'm going to close this. When a decision is made, we can spin-up a new ticket for setting it all up.
